pkg/rbd/nodeserver.go

```diff
@@ -303,3 +304,18 @@ func parseFindMntResolveSource(out string) (string, error) {
 	}
 	return "", fmt.Errorf("parseFindMntResolveSource: %s doesn't match to any expected findMnt output", out)
 }
+
+func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolumeRequest) (*csi.NodeExpandVolumeResponse, error) {
```

Comment:
This should only be valid for filesystem PVs (and don't you need to advertise ```node_expansion_required``` in ```ControllerExpandVolumeResponse``` and the ```EXPAND_VOLUME``` capability in the the node service?

Reply:
is this possible with spec 1.1.0?

Reply:
As @madhu-1 mentioned, its not possible with 1.1.0 spec. This is based on 1.1.0 implementation.